common/notification/index.ts

```diff
@@ -135,8 +135,10 @@ export async function checkReminders() {
                 throw new Error(`Notification(${reminder.id}) was supposed to contain a context when sending out reminders`);
             }
 
-            const attachmentGroup = await getAttachmentGroupByAttachmentGroupId(reminder.attachmentGroupId);
-
+            let attachmentGroup = undefined;
+            if (reminder.attachmentGroupId !== undefined) {
```

Comment:
Does Prisma really return `undefined` and not `null` for NULLable columns? Have you tested this?

Reply:
Oh true, I changed it last minute before pushing 🤦 sorry..